assets/src/components/notificationDrawer.tsx

```diff
@@ -56,66 +58,21 @@ const Content = () => {
     return <EmptyMessage />
   }
 
-  const newNotifications = notifications.filter(
+  const nUnreadNotifications = notifications.filter(
```

Comment:
Was this meant to be `newUnreadNotifications`?

Reply:
Oh, I see, it's `n` as in number - maybe `unreadNotificationsCount` or something?

Reply:
no, that's `n` as in "the number of". convention I picked up from somewhere a while back.